tsl/test/expected/compression_bgw.out

```diff
@@ -205,3 +205,66 @@ ERROR:  permission denied to start compress_chunks background process as role "n
 \set ON_ERROR_STOP 1
 RESET ROLE;
 REVOKE NOLOGIN_ROLE FROM :ROLE_DEFAULT_PERM_USER;
+\c :TEST_DBNAME :ROLE_SUPERUSER
+DROP TABLE IF EXISTS conditions CASCADE;
+NOTICE:  drop cascades to table _timescaledb_internal.compress_hyper_2_4_chunk
+CREATE TABLE conditions(
+    time TIMESTAMPTZ NOT NULL,
+    device INTEGER,
+    temperature FLOAT
+);
+SELECT * FROM create_hypertable('conditions', 'time', 'device', 30,
+       chunk_time_interval => '1 day'::interval,
+       migrate_data => true);
+ hypertable_id | schema_name | table_name | created 
+---------------+-------------+------------+---------
+             7 | public      | conditions | t
+(1 row)
+
+INSERT INTO conditions
+SELECT time, (random()*30)::int, random()*80 - 40
+FROM generate_series(NOW() - INTERVAL '90 days', NOW(), '10 min') AS time;
+CREATE VIEW conditions_summary
+WITH (timescaledb.continuous,
+      timescaledb.refresh_interval='1 minute',
+      timescaledb.max_interval_per_job = '90 days') AS
+SELECT device,
+       time_bucket(INTERVAL '1 hour', "time") AS day,
+       AVG(temperature) AS avg_temperature,
+       MAX(temperature) AS max_temperature,
+       MIN(temperature) AS min_temperature
+FROM conditions
+GROUP BY device, time_bucket(INTERVAL '1 hour', "time");
+NOTICE:  adding index _materialized_hypertable_8_device_day_idx ON _timescaledb_internal._materialized_hypertable_8 USING BTREE(device, day)
+REFRESH MATERIALIZED VIEW conditions_summary;
+ALTER TABLE conditions SET (timescaledb.compress);
+ALTER VIEW conditions_summary SET (
+      timescaledb.ignore_invalidation_older_than = '30 days'
+);
+SELECT COUNT(*) > 0 AS dropped_chunks
+  FROM drop_chunks(INTERVAL '31 days', 'conditions',
+                   cascade_to_materializations => FALSE);
+NOTICE:  making sure all invalidations for public.conditions_summary have been processed prior to dropping chunks
+ dropped_chunks 
+----------------
+ t
+(1 row)
+
+-- We need to have some chunks that are marked as dropped, otherwise
+-- we will not have a problem below.
+SELECT COUNT(*) > 0 AS has_dropped_chunks
```

Comment:
Why not simply return the count? That would be more illuminating. Or two columns, one with the count of dropped chunks and one with count of non dropped ones.

Reply:
Because the number of chunks can vary depending on timing issues. The only relevant fact for the test to succeed is that there is at least one chunk that has the flag `dropped` set to false.

Reply:
It wouldn't be a problem if the recommendations to use fixed timestamps is followed. Please try to avoid timing-dependent testing if possible. I don't see why this test should have to suffer from such issues.

Reply:
If we use fixed timestamps we are testing a code path that the user is not really using, but I switched to fixed timestamps for all the locations so that we can get 1.6.1 out.